include/deal.II/base/mpi_compute_index_owner_internal.h

```diff
@@ -77,8 +77,9 @@ namespace Utilities
          */
         class DictionaryPayLoad
           : public ConsensusAlgorithms::Process<
-              std::pair<types::global_dof_index, types::global_dof_index>,
-              unsigned int>
+              std::vector<
+                std::pair<types::global_dof_index, types::global_dof_index>>,
+              std::vector<unsigned int>>
```

Comment:
Previously, the CA algorithms took as template arguments the element types of vectors. Now they just take the overall type as template argument, which means that it is now a vector-of-something.